### DISCUSSION TITLE
Why do we have in Common Office files letters like b,c,t,d?

### DISCUSSION THREAD

**Discussion by d4h0y0sr** [DivinumOfficium/divinum-officium#3883] [Developers corner]
I understand the letters in C files like is described in technical documentation: a and p, but what about the other letters i see like b,c,t,d?

**Comment by FAJ-Munich** [comment]
It's a practicality which, as far as I have checked, is derived on how the Commune Sanctorum is actually organised in the books. For instance, there are slight variations in the Commune for Virgins, C6, depending on whether it's one or more and a martyr or not. These are then in the files C6, C6a, and C6b. In the Missale, you also find several masses for the same category of saints, like Confessor Bishop, so there is the need for even more distinguished files which are simply enumerated by the alphabet.

**Reply by d4h0y0sr** [reply]
And do you know how can i test it? i have troubles understanding how to see it on UI. Like for example i can see C6b, but i dont know how can i see C6bp. This would be helpful for testing purposes

**Reply by FAJ-Munich** [reply]
Could you please describe in more detail what you are trying to achieve? I am not quite sure what you mean by "seeing it in UI" for instance.

**Comment by mbab** [comment]
Look at `[votives]` is in `missa.dialog` file.
